src/E/E_RandomDistribution.cpp

```diff
@@ -94,6 +95,11 @@ Real ExpDistribution::nextDistribution(Real min, Real max)
 	return std::min(min + (-log(dist(engine)) / lambda), max);
 }
 
+LinearDistribution::LinearDistribution() : RandomDistribution()
+{
+
+}
+
```

Comment:
Isn't it automatically generated by the compiler?

Reply:
I also agree that it's not necessary, will be removed